docs/source/integrations/mobile/ios-fairplay-sdk.md

```diff
@@ -87,125 +86,115 @@ More information about ATS can be found in Apple’s TechNote:
+ - The `playlist_url` to correctly prepared content.
+ -  A `name` for the content (the correct content ID will be parsed out by VUDRMFairPlay SDK and will always be the last path component of the above `skd://` URI entry).
+ - A boolean value which indicates that the `Stream` object is protected with DRM. This setting would not be needed in scenarios where all content is protected, as all assets can be set as protected by default in the source code. 
+ - A `vudrm-token` which must be correctly generated for the type of instance you wish to create. Example VUDRM token type templates available are:
+	- Fairplay Rental `{"type": "r","duration_rental": 3600}`
+		- Fairplay Rental tokens may only be used to stream online content to devices running iOS 9.x or higher.
+	- Fairplay Persist `{"type": "p","duration_persist": 3600}`
+ 		- Fairplay Persist tokens may be used to stream online content and play offline (downloaded) content to devices running iOS 10.x or higher.
 
 For further information about VUDRM please contact us, or refer to our documentation:
 [https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html](https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html)
 
-## Example framework usage
-
-* Import the VUDRMFairPlay module into your view controller:
+For added convenience, VUDRMFairPlay also bubbles up notifications of errors and progress during the licensing request.
 
-	`import vudrmFairPlay`
-
-* Initialise an instance of VUDRMFairPlay:
+Errors can be intercepted as follows:
 
+    @objc func handleVudrmDidError(_ notification: Notification)
+    {
+        if let data = notification.userInfo as? [String: String]
+        {
+            for (function, error) in data
+            {
+                print("vudrmFairPlay - \(function) reported error \(error)!")
+            }
+        }
+    }
+    
+And, progress can be monitored using the following:
+   
+    @objc func handleVudrmProgressUpdate(_ notification: Notification)
+    {
+        if let data = notification.userInfo as? [String: String]
+        {
+            for (function, message) in data
+            {
+                print("vudrmFairPlay - \(function) reported progress \(message).")
+            }
+        }
+    }
+        
+## Example framework usage
+We strongly recommend referring to our example multiple asset demo application which uses a `ContentKeyManager`class to initialise an instance of VUDRMFairPlay for each stream, it then attaches the instance to an `AssetResourceLoaderDelegate`which is added to the `AssetResourceLoaderDelegateQueue`. Used with an `AssetPersistenceManager` class, the `ContentKeyManager` will also correctly handle persistence restoration with the `updateResourceLoaderDelegate` function.
+	
+The ContentKeyManager class should look like this:
+
+	import AVFoundation
+ 	import vudrmFairPlay
+ 	
+ 	class ContentKeyManager {
+    
+    /// MARK: Types.
+    
+    /// The singleton for `ContentKeyManager`
```

Comment:
I don't think inline code will work once this code is inside a code block.

Reply:
It works for me, as discussed.